### ISSUE TITLE
Add "copy address" to touchbar

### ISSUE THREAD

**Issue by joh-klein** [MacPass/MacPass#988]
**Describe the solution you'd like**
You now support the touch bar, which is really great. Thank you! I would love it, if macpass would add a fourth "button" to copy the address/url of an entry

**Comment by ryanthara** [comment]
Maybe there is not enough space for an additional button on the Touch Bar of a 13" MacBookPro.

<img width="1085" alt="Touch Bar Bild 2019-11-01 um 06 21 02" src="https://user-images.githubusercontent.com/7117072/68003929-cc42a580-fc6f-11e9-95d0-fc98d307b890.png">

**Comment by joh-klein** [comment]
That is true, but that shouldn't be an issue since you can scroll there as well (See "Sublime Text" as an example). Or alternatively use icons – either in general or when there is not enough space?

**Comment by mstarke** [comment]
Lacking a Touch Bar mac I recently tested MacPass for the first time on actual hardware and I encounters that different translations result in more or less buttons being displayed and I wasn’t able to scroll on the bar.

**Comment by unnamed777** [comment]
I'd like to have "Copy URL" too. It's useful for ssh credentials, for instance.
